common/local_utils.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for using the local_utils tool."""
+
+import os
+
+from common import logs
+from common import new_process
+
+logger = logs.Logger('local_utils')
+
+
+def local_utils_command(arguments, *args, **kwargs):
+    """Executes a local_utils command with |arguments| and returns the result.
+    """
+    return new_process.execute(arguments, *args, **kwargs)
+
+
+def cp(*cp_arguments, **kwargs):  # pylint: disable=invalid-name
+    """Executes local_utils's "cp" command with |cp_arguments| and returns the
+    returncode and the output."""
+    # Creates intermediate folders for `cp` command.
```

Comment:
`Create`

Reply:
I follow the format above.
Is there any reason why using `Executes` but `Create` here, not `Creates`?

Reply:
It's confusing and I wasn't exactly sure we do it this way myself. But from my understanding of http://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings docstrings are generally done in "descriptive style" (i.e. verbs ending in "s" and "es") while comments are done in "imperative style" e.g. "Count foos in bar".

Reply:
Actually, it's also better to explain "why" something is done rather than just what it is doing in a comment. So could you add that we create intermediate folders for `cp` because this is the behavior of `gsutil.cp` please.